src/python/WMCore/WorkQueue/WMBSHelper.py

```diff
@@ -710,8 +710,11 @@ def _addACDCFileToWMBSFile(self, acdcFile, inFileset=True):
             wmbsFile.addRun(run)
 
         dbsFile = self._convertACDCFileToDBSFile(acdcFile)
-        self._addToDBSBuffer(dbsFile, checksums, acdcFile["locations"])
 
+        if "/store/unmerged/" not in dbsFile["LogicalFileName"] or dbsFile["ParentList"]:
```

Comment:
shouldn't it be `if not dbsFile["LogicalFileName"].startswith("/store/unmerged/")...`

Reply:
I can changed it but is there case if /store/unmerged/ is in the string but actually not a unmerged file?